cmd/kubeadm/app/constants/constants.go

```diff
@@ -171,6 +171,11 @@ const (
 
 	// DefaultCIImageRepository points to image registry where CI uploads images from ci-cross build job
 	DefaultCIImageRepository = "gcr.io/kubernetes-ci-images"
+
+	// CoreDNS points to the version of CoreDNS
```

Comment:
hmm... what version?
Maybe just `CoreDNS defines a variable used internally when referring to the CoreDNS addon for a cluster`